pipeline/beam_tables.py

```diff
@@ -372,6 +374,22 @@ def _get_full_table_name(self, table_name: str) -> str:
     """
     return self.project + ':' + table_name
 
+  def _compare_sets(self, table_set: set, gcs_set: set) -> None:
```

Comment:
add a docstring (see elsewhere in this file) explaining the function, types and especially the raised errors